src/views/AbstractView.ts

```diff
@@ -149,7 +149,7 @@ abstract class AbstractView {
         entity =>
           entity.entity_id.startsWith(domain + ".")
           && !entity.hidden_by
-          && !Helper.strategyOptions.card_options?.entity_id.hidden
+          && !Helper.strategyOptions.card_options?.entity_id?.hidden
```

Comment:
Actually, I think `entity_id` will always be undefined, because it should be `entity.entity_id`.

```suggestion
          && !Helper.strategyOptions.card_options?.[entity.entity_id]?.hidden
```

Please verify my suggested change.

Reply:
Yes, indeed. When I wrote the code I didn't understand the purpose, now I know that it serves to create an array the domain switches on and off when using the on / off buttons on the domain view